config/standalone.example.toml

```diff
@@ -152,6 +152,36 @@ auto_flush_interval = "1h"
 # Global write buffer size for all regions.
 global_write_buffer_size = "1GB"
 
+# Mito engine options
+[[region_engine]]
+[region_engine.mito]
+# Number of region workers
+num_workers = 8
+# Request channel size of each worker
+worker_channel_size = 128
+# Max batch size for a worker to handle requests
+worker_request_batch_size = 64
+# Number of meta action updated to trigger a new checkpoint for the manifest
+manifest_checkpoint_distance = 10
+# Manifest compression type
+manifest_compress_type = "Uncompressed"
```

Comment:
Do we accept lower case for compression type? It's better to use lowercase.

Reply:
> Do we accept lower case for compression type?

I'm not sure, maybe not. I copied the sample config from `datanode.example.toml`. It reuses the enum from `common-datasource` crate so it's out of this PR's topic. We can leave an issue for it.